integration-tests/networks/known_networks.go

```diff
@@ -199,7 +199,7 @@ var (
 
 	OptimismMainnet blockchain.EVMNetwork = blockchain.EVMNetwork{
 		Name:                      "Optimism Mainnet",
-		ClientImplementation:      blockchain.MetisClientImplementation, // Optimism Bedrock has not been released yet, use Metis for Legacy Tx Support
```

Comment:
@kalverra I've changed this since Optimism Mainnet has migrated to Bedrock. Let me know if this works and whether we need to make another change or not.

Reply:
That should be all you need